swig/swigResqml2Include.i

```diff
@@ -2208,6 +2223,9 @@ namespace RESQML2_NS
 		bool isAChronoStratiRank() const;
 		StratigraphicUnitInterpretation* getSubjectOfContact(unsigned int contactIndex) const;
 		StratigraphicUnitInterpretation* getDirectObjectOfContact(unsigned int contactIndex) const;
+		std::vector<RESQML2_NS::StratigraphicOccurrenceInterpretation *> getStratigraphicOccurrenceInterpretationSet() const;
```

Comment:
Should propbably define and use macro for getters.

Reply:
Done.